docs/src/tutorial/accept_loop.md

```diff
@@ -25,7 +25,7 @@ type Result<T> = std::result::Result<T, Box<dyn std::error::Error + Send + Sync>
 
 Now we can write the server's accept loop:
 
-```rust,edition2018
```

Comment:
It seems `edition2018` should be kept, as it's causing the tests to fail https://travis-ci.com/async-rs/async-std/jobs/237051125

Reply:
Yep, my bad 👍